src/angular-business/notification/notification/notification.component.html

```diff
@@ -1,35 +1,10 @@
-<span class="sbb-notification-icon-wrapper">
-  <ng-container *ngTemplateOutlet="icon"></ng-container>
-</span>
-
-<div class="sbb-notification-content">
-  <div class="sbb-notification-content-message">
-    <ng-content></ng-content>
-  </div>
-  <div *ngIf="jumpMarks">
-    <span *ngFor="let jumpMark of jumpMarks; let isLast = last" class="sbb-notification-jump-mark">
-      <a (click)="scrollTo($event, jumpMark)">{{ jumpMark.title }}</a>
-      <ng-container *ngIf="!isLast"> / </ng-container>
-    </span>
-  </div>
+<div class="sbb-notification-content-message" [innerHTML]="config.message"></div>
```

Comment:
There seems to be a massive misunderstanding. The existing notification component should not be changed and remain as it was.
For the Toast Notification a new component should be created (e.g. SimpleNotification).